website/docs/intro/core-workflow.mdx

```diff
@@ -239,7 +239,13 @@ terraform {
     hostname = "app.terraform.io" # Optional; defaults to app.terraform.io
 
     workspaces {
-      tags = ["networking", "source:cli"]
+      tags = {
+        layer = "networking"
+        source = "cli"
+      }
+      # For terraform versions below 1.10, you must specify key-only tags
```

Comment:
nit: consider a newline above the comment so the sections are not squished together